validation/jinja2/validation/_base.html

```diff
@@ -13,6 +13,10 @@
     margin: 1em auto;
   }
 
+  .bg-dark {
+    background-color: #143B6D !important;
```

Comment:
What kind of specificity conflict requires the use of `!important`?

Reply:
`.bg-dark` is a bootstrap styles class that I wanted to change a little. There will be another PR at the end of all this with properly implemented styles in their own file.